### PR TITLE
TW-891: Update Twake Logo in Web app

### PR DESCRIPTION
**Pull request by imGok**
#891 
image
image
image
image
image

### CODE REVIEW

**Comment by nqhhdev** [comment]
- Sorry Julian, my bad. Pls update splash screen also

Screenshot 2023-11-03 at 21 00 13

**Review by hoangdat — COMMENTED** [review]
How about the app name @imGok

**Review by drminh2807 — APPROVED** [review]

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -35,7 +35,7 @@
     <uses-sdk
             tools:overrideLibrary="io.wazo.callkeep, net.touchcapture.qr.flutterqr, com.cloudwebrtc.webrtc, org.webrtc, com.it_nomads.fluttersecurestorage, com.pichillilorenzo.flutter_inappwebview, com.example.video_compress, com.otaliastudios.transcoder, com.otaliastudios.opengl, com.kineapps.flutter_file_dialog, com.llfbandit.record, com.pravera.flutter_foreground_task"/>
     <application
-        android:label="Twake"
+        android:label="Twake Chat"
```
Comment (imGok) [inline android/app/src/main/AndroidManifest.xml:38]: @hoangdat is it impacting something ?

Comment (hoangdat) [reply android/app/src/main/AndroidManifest.xml:38]: no problem

File: assets/favicon.png
Comment (hoangdat) [inline assets/favicon.png:1]: please use the same dimension logo

File: assets/images/ic_twake_logo.svg
Comment (hoangdat) [inline assets/images/ic_twake_logo.svg:1]: duplicted with ic_twake_image_logo.svg?

File: assets/logo.png
Comment (hoangdat) [inline assets/logo.png:1]: should use the same dimension

File: web/icons/Icon-386.png
Comment (hoangdat) [inline web/icons/Icon-386.png:1]: why we need to add this size?

**Review by hoangdat — COMMENTED** [review]
I think this change still miss for Android/iOS

File: assets/images/ic_twake_logo.svg
Comment (imGok) [reply assets/images/ic_twake_logo.svg:1]: size is different but they are the same svg yes

File: web/icons/Icon-386.png
Comment (imGok) [reply web/icons/Icon-386.png:1]: deleted it

File: android/app/src/main/AndroidManifest.xml
Comment (hoangdat) [inline android/app/src/main/AndroidManifest.xml:1]: my god, a lot of format. Please respect the current status, it belongs to Android world

File: assets/favicon.png
Comment (hoangdat) [inline assets/favicon.png:1]: please ask designer for the border version too

File: assets/logo.png
Comment (hoangdat) [inline assets/logo.png:1]: why you delete it? Any impaction

File: ios/Runner/Info.plist
Comment (hoangdat) [inline ios/Runner/Info.plist:1]: no, iOS world

File: linux/packaging/appimage/make_config.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 display_name: Twake
 
-icon: assets/logo.png
+icon: assets/favicon.png
```
Comment (hoangdat) [inline linux/packaging/appimage/make_config.yaml:3]: fav must be the fav, logo must be the logo. Maybe fav is very small size

File: ios/Runner/Info.plist
Comment (imGok) [reply ios/Runner/Info.plist:1]: I used this tool to rename the app https://pub.dev/packages/rename_app

File: assets/logo.png
Comment (imGok) [reply assets/logo.png:1]: it was a duplicate to favicon.png

File: linux/packaging/appimage/make_config.yaml (hunk 2 above)
Comment (imGok) [reply linux/packaging/appimage/make_config.yaml:3]: I'll wait for rounded icon to add logo again

File: assets/favicon.png
Comment (imGok) [reply assets/favicon.png:1]: done

File: android/app/src/main/AndroidManifest.xml
Comment (imGok) [reply android/app/src/main/AndroidManifest.xml:1]: I kept the format and just renamed twake

File: web/splash/img/dark-1x.png
Comment (hoangdat) [inline web/splash/img/dark-1x.png:1]: please try use the same dimension

Comment (hoangdat) [reply web/splash/img/dark-1x.png:1]: try to use the `export` feature of figma. you can specific the size you want. I succeeded.

Screenshot 2023-11-08 at 08 29 27

Comment (imGok) [reply web/splash/img/dark-1x.png:1]: thank you I didn't know I could type in this box

**Pull request merged**